lms/djangoapps/certificates/views/webview.py

```diff
@@ -129,11 +129,21 @@ def _update_certificate_context(context, course, user_certificate, platform_name
         platform_name=platform_name
     )
 
+    # edx-417: this is used in SOA, replaces document_title there
+    context['document_title_1_a'] = _("UC San Diego").upper()
+    context['document_title_1_b'] = _("Online").upper()
+    context['document_title_2'] = _("Statement of Accomplishment").upper()
+
     # Translators:  This text fragment appears after the student's name (displayed in a large font) on the certificate
     # screen.  The text describes the accomplishment represented by the certificate information displayed to the user
-    context['accomplishment_copy_description_full'] = _("successfully completed, received a passing grade, and was "
-                                                        "awarded this {platform_name} "
-                                                        "Statement of Accomplishment in ").format(platform_name=platform_name)
+    # edx-417
+    context['accomplishment_copy_description_full'] = _("has successfully completed an offering of")
+    context['accomplishment_disclaimer'] = _("A Statement of accomplishment acknowledges that a {platform_name} " +
```

Comment:
if we want italic font on _`Onilne`_, do we want to replace the platform name?

Reply:
i think we can skip the italics in the disclaimer - thanks @tasawernawaz !